pincer/objects/message/user_message.py

```diff
@@ -567,7 +567,7 @@ def clean_content(self) -> str:
             new_content = re.sub(pattern, r"\1", new_content)
 
         return re.sub(
-            re.compile(r"(.*?)```[a-zA-Z]+(\s*)+\n((?:.|\s)*?)```"),
```

Comment:
not sure to understand the change (not strong at regex ;-;) mind make a brief explanation?

Reply:
Basically changing from exactly 1 `\n` to at least 1 `\n`

Reply:
You thought I understand regex?